src/client/routes/Portfolio/index.tsx

```diff
@@ -138,80 +170,25 @@ export const Portfolio = () => {
 
       {selectedRole === BORROWER_POSITION_ROLE && (
         <StyledBorrowerContainer>
-          <PositionsTable positions={borrowerPositions} displayLine={true} />
+          <PositionsTable
+            borrower={portfolioAddress!}
+            lender={undefined}
+            positions={borrowerPositions}
+            displayLine={true}
+          />
         </StyledBorrowerContainer>
       )}
 
       {selectedRole === LENDER_POSITION_ROLE && (
         <StyledBorrowerContainer>
-          <PositionsTable positions={lenderPositions.map((id) => allPositions[id])} displayLine={true} />
+          <PositionsTable
+            borrower={undefined}
```

Comment:
Dont have to define lender/borrower if they are undefined. just delete line and its the same thing